addons/api/package.json

```diff
@@ -44,7 +44,7 @@
     "broccoli-asset-rev": "^3.0.0",
     "doctoc": "^2.2.0",
     "ember-auto-import": "^1.12.0",
-    "ember-cli": "~4.1.0",
```

Comment:
Good question. I do not remember why. But now you mentione it, `ember-cli` is quite important part of our stack. So to prevent any issues with resolving a major version, should we allow just path version? let me know your thoughts @ZedLi

Reply:
I'm actually fine with changing it to allow minor versions but I wasn't sure if there was a reason for it or was it something that wasn't intentional.

Either way a major version won't get updated with either `^` or `~` so I'm fine either way!